citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '7.4-1'
+default_version = '7.4-3'
```

Comment:
It'd be nice if there was only one of these, but in the absence of that we could maybe put a comment in both files pointing out that you need to change the version in the other one as well.

Reply:
I can add a comment, but, I think if the appveyor were not failing for some unrelated things, we could have realized issues due to version upgrades.

I think this is a great example that we should not let appveyor or any other CI to be in failing state.

Reply:
The comment is just to save everyone who forgets this an hour.

Reply:
I was going to propose making `appveyor ` not optional for merging into master. Otherwise, we'll likely to get into the similar situation again?

(I can the comment as well those are not dependent 😄)

Reply:
> I was going to propose making appveyor not optional for merging into master. Otherwise, we'll likely to get into the similar situation again?

Yep, seems fine.

> (I can add the comment as well those are not dependent 😄)

Yep, it's just kind of hard to figure out that the schema is stale (we should maybe add it to the output in `multi_extension`)

Reply:
I've opened https://github.com/citusdata/citus/pull/2173 to help us remember to update `citus.control`.